### ISSUE TITLE
The detection score you provide is not normalized?

### ISSUE THREAD

**Issue by a43992899** [hailanyi/3D-Multi-Object-Tracker#6]
I find that the detection score you use is not normalized (not in the range of [0, 1]). Which layer or where did you extract those scores from? Are those score the logits before softmax?

`Car -1 -1 -8.0728 299.5648 166.8018 457.8472 299.3745 2.0278 1.8372 4.4899 -4.5203 1.9349 13.4348 -8.3923 6.8229
Cyclist -1 -1 -8.3277 743.3280 157.8975 945.0159 374.0000 1.7928 0.7382 1.7908 1.6935 1.6931 5.7445 -8.0567 6.3583
Pedestrian -1 -1 -8.9000 1086.0386 169.0511 1223.0736 325.3153 1.7003 0.7000 1.0835 6.3164 1.6586 8.4887 -8.2776 3.9818
Car -1 -1 -2.0002 0.0000 198.3879 22.4742 238.5690 1.4643 1.6507 3.9074 -26.4072 2.5741 29.8174 -2.7207 -0.4804
Car -1 -1 -7.6264 284.9278 190.2203 347.9479 230.2715 1.5499 1.6856 4.1383 -12.9598 2.3717 31.9821 -8.0090 -0.7022
Car -1 -1 -4.6372 136.7996 189.1622 176.2292 219.5773 1.5594 1.6092 4.0767 -26.0653 2.5465 41.5085 -5.1953 -1.3563
Car -1 -1 -2.5441 0.0000 195.0718 83.9934 227.2724 1.4837 1.6807 3.8615 -28.5618 2.6095 35.5869 -3.2169 -2.0098
Car -1 -1 -4.4626 347.0251 177.7920 392.8701 208.4988 1.6766 1.6258 3.8540 -13.9505 1.9773 42.0110 -4.7817 -2.1874`

**Comment by qq852518421** [comment]
using sigmoid in trajectory 252,254

**Comment by hailanyi** [comment]
The detection score is not normalized before put into the tracker. For the OpenPCDet, you can set the OUTPUT_RAW_SCORE=Ture in the detector [config file](https://github.com/open-mmlab/OpenPCDet/blob/master/tools/cfgs/kitti_models/voxel_rcnn_car.yaml) to obtain the detection results.

**Comment by Yuchen-Tao** [comment]
Hi, does it mean that the tracker is designed only for unnormalized scores? I see that the prediction scores are initialized as 1 for example.
